var/spack/repos/builtin/packages/py-questionary/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyQuestionary(PythonPackage):
+    """Questionary is a Python library for effortlessly building pretty
+    command line interfaces."""
+
+    homepage = "https://github.com/tmbo/questionary"
+    pypi     = "questionary/questionary-1.9.0.tar.gz"
+
+    version('1.9.0', sha256='a050fdbb81406cddca679a6f492c6272da90cb09988963817828f697cf091c55')
+
+    depends_on('python@3.6:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@3.6:3.9.999', type=('build', 'run'))
```

Reply:
Fixed (I went for 3.999 because there could also be a Python 3.10 at some point, right?)

Reply:
The setup.py explicitly says it won't work for 3.10

Reply:
Sorry, totally missed that. I checked the source repository (https://github.com/tmbo/questionary), which states `python ^3.6` in its `pyproject.toml` (which should mean `@3.6:3.999`). Not sure how they generate the `setup.py` from that. I have updated the version range to be consistent with the `setup.py`.